src/packageAPI.ts

```diff
@@ -196,9 +196,41 @@ async function getDependencies(dependencies: string[], rateLimiter: PackageRateL
 		return depMap
 	}
 
+async function getDependenciesNPMSio(dependencies: string[], rateLimiter: PackageRateLimiter, baseUrl: string): Promise<Map<string, { version: string; link: string; }>> {
+		let depMap: Map<string, { version: string, link: string, languageVersion?: string }> = new Map()
+
+		await rateLimiter.npm.tokenBucket.waitForTokens(1)
+		// TODO: limit calls to 250 packages
+		const requestOptions = {
+		method: 'POST',
+		headers: {
+			"Accept": "application/json",
+			"Content-Type": "application/json"
+			},
+			body: JSON.stringify(dependencies)
+		};
+		// const depList: {name: string, data: {version: string, link: string}}[] = []
+
+		await fetch("https://api.npms.io/v2/package/mget", requestOptions)
+		.then(response => response.json())
+		.then(response => {
+			const tempList: any  = []
+			for (const dependency in response) {
+				// @reteppeter how are you getting _nodeVersion from npm,
+				// for example the node version for @sanity/block-content-to-react is not defined in package-lock
```

Comment:
We don't get Node versions from npm, we only get them from the package.json's of packages within the organisation.

Reply:
> This will trigger a new crawling loop every time there is a new webhook event, right?

This will trigger a new crawling loop every time you call `handleGitHubWebhookPushEvents` or `handleGitHubWebhookRepositoryEvents`.

Race condition will occur if they are not called(and awaited) sequentially.